lib/usage.js

```diff
@@ -151,20 +153,38 @@ module.exports = function (yargs, y18n) {
       })
     })
 
-    var switches = keys.reduce(function (acc, key) {
-      acc[key] = [ key ].concat(options.alias[key] || [])
-      .map(function (sw) {
-        return (sw.length > 1 ? '--' : '-') + sw
+    // populate 'Options:' group with any keys that have not
+    // explicitly had a group set.
+    addUngroupedKeys(keys, options.alias, groups)
+
+    // display 'Options:' table along with any custom tables:
+    Object.keys(groups).forEach(function (groupName) {
+      if (!groups[groupName].length) return
+
+      ui.div(__(groupName))
+
+      // if we've grouped the key 'f', but 'f' aliases 'foobar',
+      // normalizedKeys should contain only 'foobar'.
+      var normalizedKeys = groups[groupName].map(function (key) {
+        if (~aliasKeys.indexOf(key)) return key
+        for (var i = 0, aliasKey; (aliasKey = aliasKeys[i]) !== undefined; i++) {
+          if (~(options.alias[aliasKey] || []).indexOf(key)) return aliasKey
+        }
+        return key
       })
-      .join(', ')
 
-      return acc
-    }, {})
+      // actually generate the switches string --foo, -f, --bar.
+      var switches = normalizedKeys.reduce(function (acc, key) {
+        acc[key] = [ key ].concat(options.alias[key] || [])
+        .map(function (sw) {
+          return (sw.length > 1 ? '--' : '-') + sw
+        })
+        .join(', ')
```

Comment:
`standard` doesn't seem to care, but can we indent these 4 lines (since they continue the value end of the `acc[key]` assignment)?